src/buttons/Button.js

```diff
@@ -181,7 +207,7 @@ Button.propTypes = {
   fontFamily: PropTypes.string,
 };
 
-const styles = StyleSheet.create({
+const stylesObject = {
```

Comment:
The point was to access nested properties, as I mentioned in the PR description - I needed to calculate dimensions based on font size and I also pick correct color based on what the user specified, this is then passed to baseFont const. With StyleSheet you can't access nested properties (like styles.text.fontSize), the only way is to have styles.textFontSize and to me that can add up, but when you use an actual object, you can access them. That's why I wrote in the description that probably you won't agree with that, but it helped me while prototyping. It was just a proposition, we can revert to having only StyleSheet and using keys like styles.textFontSize for this.